### ISSUE TITLE
Size mismatch error when loading a pretrained model

### ISSUE THREAD

**Issue by jackywangtj66** [apple/ml-cvnets#87]
I tried to load a pretrained mobilevit model downloaded from model zoo for fine-tuning on a classification task with 6 labels. I do have excluded the last layer:
![image](https://github.com/apple/ml-cvnets/assets/63412857/84f538ae-2b9d-4662-ac8d-a3c9d212a6d5)

But I still encountered the following error:
2023-08-16 15:05:22 - ERROR   - Unable to load pretrained weights from ./TrainedModels/Models/MobileVitv1/mobilevit_xxs.pt. Error: Error(s) in loading state_dict for MobileViT:
        size mismatch for classifier.fc.weight: copying a param with shape torch.Size([1000, 320]) from checkpoint, the shape in current model is torch.Size([6, 320]).
        size mismatch for classifier.fc.bias: copying a param with shape torch.Size([1000]) from checkpoint, the shape in current model is torch.Size([6]).. Exiting!!!

Do you have any idea?

**Comment by jackywangtj66** [comment]
Found the problem myself.
Resume_exclude_scopes should be under 'model' instead of 'model.classification'.
Suggestion: would you consider putting the scopes arguments under 'pretrained'? Since they will be useless without a pretrained model.

**Comment by hrx20000209** [comment]
Hi, where did you find the pretrain models? Form huggingface?

**Comment by farzadab** [comment]
The model zoo should have links to all models: https://apple.github.io/ml-cvnets/en/general/README-model-zoo.html
